mllib-local/src/main/scala/org/apache/spark/ml/linalg/Matrices.scala

```diff
@@ -153,6 +153,86 @@ sealed trait Matrix extends Serializable {
    */
```

Comment:
BTW, it's nice to have return type in public method. Can you add `Unit` as return type?